zephyr/port/wifi/wifi_stubs.c

```diff
@@ -0,0 +1,1222 @@
+#include "utils/includes.h"
+#include "utils/common.h"
+
+/* ################################# */
```

Comment:
can we replace this comment line with something more descriptive if that is needed?